.eslintrc.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
     "brace-style": "off",
     'capitalized-comments': "off",
     "class-methods-use-this": "off",
+    "comma-dangle": "off",
```

Comment:
I don't think this change ought to be incorporated in the PR, unless you want to bog this down with a style discussion.

Reply:
i'm personally neutral on this rule.  not wild about the way it reads when i'm looking at code, but it does lead to slightly nicer diffs because it means just a little bit less churn, since the last list entry is no longer special in terms of not having a comma after it.

would agree w/ @jcoyne's point that it warrants discussion, since i suspect some might want this rule?  when i ran into it in the past, i just played along with the restriction.

Reply:
I can up the warnings and do it separately.  It actually creates invalid json for our inline json and for json objects.